src/views/sets.js

```diff
@@ -488,6 +488,106 @@ module.exports = [
           },
         ],
       },
+      {
+        name: 'Hunter Specific Sets',
+        description: 'Other sets of Purple Gear obtained from Engrams',
+        sections: [
+          {
+            title: 'Dead End Cure 2.1',
+            items: [
+              3125909492,
+              1998314509,
+              316000947,
+              2669113551,
```

Comment:
I think this is the cloak of this set: 2521426922

Reply:
That came up in the search because of the flavor text, but given that it had a different name, I excluded it. I could definitely add it.

Reply:
Like I said, it's a guess. I think that except for the "Be thy XXX" class items, all of them are part of sets.

Reply:
idk where the other ones belong, though